Examples/IcmpFileTransfer/IcmpFileTransfer-pitcher.cpp

```diff
@@ -36,12 +36,13 @@ void usleep(__int64 usec)
 	LARGE_INTEGER ft;
 
 	ft.QuadPart = -(10 * usec);  // Convert to 100 nanosecond interval, negative value indicates relative time
-
+	                             // NULL is Used for WindowAPIs and no need to change to nullptr for C++11
```

Comment:
Still misaligned. Maybe you can add a newline before your comment. Probably editors are trying to align with line 38.